src/ast.rs

```diff
@@ -7297,6 +7310,29 @@ impl Parse for SpecialName {
                     tail,
                 ))
             }
+            b"GT" => {
+                match tail.peek() {
+                    Some(b'n') => {
+                        let tail = consume(b"n", tail).unwrap();
+                        let (base, tail) = Encoding::parse(ctx, subs, tail)?;
+                        Ok((
+                            SpecialName::NonTransactionClone(Box::new(base)),
+                            tail,
+                        ))
+                    }
+                    // Different letters stand for different types of transactional cloning,
+                    // so they should be treated the same.
```

Comment:
Maybe something more like "Different letters could stand for different types of transactional cloning, but for now, treat them all the same"? Also note that GTt is the only one observed thus far.